profile/gallery-benchmarks.md

```diff
@@ -0,0 +1,47 @@
+# Benchmarks gallery
+
+To reproduce/modify a benchmark,
+    
+1. clone the benchmark repository and ``cd`` into it,
+
+```shell
+$ git clone https://github.com/benchopt/{REPOSITORY_NAME}.git
+$ cd {REPOSITORY_NAME}
+```
+
+2. install ``benchopt`` to use its API/CLI,
+
+```shell
+$ pip install -U benchopt
+```
+    
+3. run the following command to reproduce the benchmark,
+
+```shell
+$ benchopt run .
+```
+
+Et Voilà! Refer to [``Benchopt`` API](https://benchopt.github.io/cli.html#benchopt-run) 
+to explore the different options for running a benchmark.
+
+
+## Available benchmarks
+
+- [Convolutional sparse coding](https://api.github.com/repos/benchopt/benchmark_csc) ![Test status](https://github.com/benchopt/benchmark_csc/actions/workflows/main.yml/badge.svg)
+- [The elastic net problem](https://api.github.com/repos/benchopt/benchmark_elastic_net) ![Test status](https://github.com/benchopt/benchmark_elastic_net/actions/workflows/main.yml/badge.svg)
+- [Glm model](https://api.github.com/repos/benchopt/benchmark_glm) ![Test status](https://github.com/benchopt/benchmark_glm/actions/workflows/main.yml/badge.svg)
+- [No description.](https://api.github.com/repos/benchopt/benchmark_huber_l2) ![Test status](https://github.com/benchopt/benchmark_huber_l2/actions/workflows/main.yml/badge.svg)
```

Comment:
this name should be updated

Reply:
To generate the ``md`` file

1. I go through all ``benchopt`` repository
2. I keep the ones with a name that starts with _benchmark_
    (all benchmark repos start with benchmark)
3. I fetch the description of these repos and process them
4. I generate the item: 
```python
f"- [{description}]({repo.url}) "
f"![Test status](https://github.com/benchopt/{repo.name}/actions/workflows/main.yml/badge.svg)\n")
```


Therefore, all items that with _[No description.]_ are benchmark repositories with no repository description. (See examples of [benchmark_huber_l2](https://github.com/benchopt/benchmark_huber_l2)

![example-benchmark-without-description](https://user-images.githubusercontent.com/65614794/174500144-6d7efa1f-3525-45bc-b2d7-15e07ba8da0b.png)


I think standardizing the benchmark repositories (name, and description) would reduce drastically the cost of maintenance (as one can generate automatically the benchmarks gallery) and would pay off in the long run.

Reply:
I have filled the descriptions in the 3 repos